### PR TITLE
Create .appveyor.yml

### PR DESCRIPTION
**Pull request by jkuruvilla**
- Added AppVeyor CI for windows builds.
- Tested on local repository
- Successful build report: https://ci.appveyor.com/project/jkuruvilla/iota-lib-rs-6j3j0

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+environment:
+  matrix:
+    - channel: nightly
+      target: x86_64-pc-windows-msvc
+    - channel: nightly
+      target: i686-pc-windows-msvc
+    - channel: nightly
+      target: x86_64-pc-windows-gnu
+    - channel: nightly
+      target: i686-pc-windows-gnu
```
Comment (njaremko) [inline .appveyor.yml]: I don't intend to support 32 bit windows, so might as well take this out.

**Comment by njaremko** [comment]
Might as well add a badge to the README while you're at it

File: .appveyor.yml (hunk 1 above)
Comment (jkuruvilla) [reply .appveyor.yml]: Done

Comment (njaremko) [reply .appveyor.yml]: Thanks, did you see my other comment?

Comment (jkuruvilla) [reply .appveyor.yml]: Replied to the other comment as well :)

**Comment by njaremko** [comment]
Merged. Thanks

**Pull request merged**